<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="loginStack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>600</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="loginPage">
     <widget class="QWidget" name="loginElement" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>251</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#loginElement{
	background:#DCDCDC
}</string>
      </property>
      <widget class="QLineEdit" name="userNameTxT">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Username*</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordTxt">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password*</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loginButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>250</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#loginButton{
	background:#20B2AA;
	color:black;
	border:none;
	border-radius:6px;
}
#loginButton:hover{
	background:#008080
}</string>
       </property>
       <property name="text">
        <string>Sign In</string>
       </property>
      </widget>
      <widget class="QLabel" name="loginIcon">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>49</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Resource.qrc">:/picture/login.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="loginTxt">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>90</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">#loginTxt{
	color:black;
}</string>
       </property>
       <property name="text">
        <string>Sign In</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="loginPicture">
      <property name="geometry">
       <rect>
        <x>251</x>
        <y>0</y>
        <width>649</width>
        <height>600</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resource.qrc">:/picture/company.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mainPage">
     <widget class="QWidget" name="sideBar" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>271</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#sideBar{
	background:#003366
}</string>
      </property>
      <widget class="QLabel" name="avatarSidebar">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>101</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Resource.qrc">:/picture/avatar.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="title">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>261</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#title{
	color:#00FF00;
}</string>
       </property>
       <property name="text">
        <string> Employee Management System</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="nameUser">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#nameUser{
	color:white;
}</string>
       </property>
       <property name="text">
        <string>WELCOME DUC DO</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="searchButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>200</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#searchButton{
	border:none;
	text-align:left;
	padding-left:20px;
	background:#6666FF;
	color:white;
	border-left:6px solid #97FFFF;
}</string>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
       <property name="icon">
        <iconset resource="Resource.qrc">
         <normaloff>:/picture/SearchIcon.png</normaloff>:/picture/SearchIcon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="addButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>241</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#addButton{
	background:#003366;
	border:none;
	color:white;
	text-align:left;
	padding-left:26px;
}
#addButton:hover{
	border:none;
	text-align:left;
	padding-left:26px;
	background:#6666FF;
	color:white;
}</string>
       </property>
       <property name="text">
        <string>Add Employee</string>
       </property>
       <property name="icon">
        <iconset resource="Resource.qrc">
         <normaloff>:/picture/AddIcon.png</normaloff>:/picture/AddIcon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="updateButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>280</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#updateButton{
	background:#003366;
	border:none;
	color:white;
	text-align:left;
	padding-left:26px;
}
#updateButton:hover{
	border:none;
	text-align:left;
	padding-left:26px;
	background:#6666FF;
	color:white;
}</string>
       </property>
       <property name="text">
        <string>Update Employee Record</string>
       </property>
       <property name="icon">
        <iconset resource="Resource.qrc">
         <normaloff>:/picture/UpdateUser.png</normaloff>:/picture/UpdateUser.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="deleteButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>320</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#deleteButton{
	background:#003366;
	border:none;
	color:white;
	text-align:left;
	padding-left:26px;
}
#deleteButton:hover{
	border:none;
	text-align:left;
	padding-left:26px;
	background:#6666FF;
	color:white;
}</string>
       </property>
       <property name="text">
        <string>Delete Employee Record</string>
       </property>
       <property name="icon">
        <iconset resource="Resource.qrc">
         <normaloff>:/picture/DeleteUser.png</normaloff>:/picture/DeleteUser.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="logoutButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>440</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#logoutButton{
	background:#003366;
	border:none;
	color:white;
	text-align:left;
	padding-left:26px;
}
#logoutButton:hover{
	border:none;
	text-align:left;
	padding-left:26px;
	background:#6666FF;
	color:white;
}</string>
       </property>
       <property name="text">
        <string>Logout</string>
       </property>
       <property name="icon">
        <iconset resource="Resource.qrc">
         <normaloff>:/picture/logout.png</normaloff>:/picture/logout.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="techButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>360</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#techButton{
	background:#003366;
	border:none;
	color:white;
	text-align:left;
	padding-left:26px;
}
#techButton:hover{
	border:none;
	text-align:left;
	padding-left:26px;
	background:#6666FF;
	color:white;
}</string>
       </property>
       <property name="text">
        <string>Technologies Used</string>
       </property>
       <property name="icon">
        <iconset resource="Resource.qrc">
         <normaloff>:/picture/techIcon.png</normaloff>:/picture/techIcon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="aboutButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>400</y>
         <width>271</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#aboutButton{
	background:#003366;
	border:none;
	color:white;
	text-align:left;
	padding-left:26px;
}
#aboutButton:hover{
	border:none;
	text-align:left;
	padding-left:26px;
	background:#6666FF;
	color:white;
}</string>
       </property>
       <property name="text">
        <string>About App</string>
       </property>
       <property name="icon">
        <iconset resource="Resource.qrc">
         <normaloff>:/picture/aboutIcon.png</normaloff>:/picture/aboutIcon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
